src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $main-card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: scale 0.3s;
+
+  &:hover {
+    scale: 1.2;
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 40px 0 0;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    letter-spacing: 0%;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the `letter-spacing` property should use relative units like `em` or `%` only if it's not equal to `0`. Since it's `0%` here, this declaration can be removed.

Reply:
And here